overlord/devicestate/users_test.go

```diff
@@ -178,7 +178,7 @@ func (s *usersSuite) TestCreateUser(c *check.C) {
 	c.Check(user.Email, check.Equals, s.userInfoExpectedEmail)
 	c.Check(user.Macaroon, check.NotNil)
 	c.Check(addUserCalled, check.Equals, true)
-	c.Check(user.Expiration, check.Equals, expectedExpiration)
+	c.Check(user.Expiration.Equal(expectedExpiration), check.Equals, true)
```

Comment:
aren't there more places that could use this change?

Reply:
Ah yeah right, updated the remaining places.